bundles/CoreBundle/src/Command/Asset/GenerateChecksumCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Command\Asset;
+
+use Pimcore\Console\AbstractCommand;
+use Pimcore\Db\Helper;
+use Pimcore\Model\Asset;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @internal
+ */
+class GenerateChecksumCommand extends AbstractCommand
+{
+    protected function configure(): void
+    {
+        $this
+            ->setName('pimcore:assets:generate-checksums')
+            ->setDescription('Re-generates checksum for specific or all assets')
```

Comment:
Should we maybe add an option to calculate the checksum for all assets that doesn't have one set?

Reply:
Add option `missing-only` to generate checksum for assets not having one.